Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -2013,4 +2025,26 @@ public boolean isCalendarButtonShowing() {
     public FileformatInterface getFileformat() {
         return rdf;
     }
+
+    // TODO:
+    // replace calls to this function with "/pages/processFromTemplate" once we have completely
+    // switched to the new frontend pages
+    private String redirectToProcessFromTemplateEdit() {
+        try {
+            String referer = FacesContext.getCurrentInstance().getExternalContext().getRequestHeaderMap()
+                    .get("referer");
+            String callerViewId = referer.substring(referer.lastIndexOf("/") + 1);
+            if (!callerViewId.isEmpty()
+                    && (callerViewId.contains("projects.jsf"))) {
+                return PROCESS_FROM_TEMPLATE_PATH + "?" + REDIRECT_PARAMETER;
+            } else {
+                return PROCESS_FROM_TEMPLATE_PATH_OLD + "?" + REDIRECT_PARAMETER;
+            }
+        } catch (NullPointerException e) {
+            // This NPE gets thrown - and therefore must be caught - when "ProzessverwaltungForm" is
+            // used from it's integration test
+            // class "ProzessverwaltungFormIT", where no "FacesContext" is available!
```

Comment:
On copy and pasting code from other classes, please adjust the comments. You added this to class `ProzesskopieForm` but your comment is about class `ProzessverwaltungFormIT` two times.